src/main/resources/io/jenkins/plugins/oidc_provider/config/ClaimTemplate/help-requiredEnvVars.html

```diff
@@ -0,0 +1,3 @@
+<div>
+    A space separated list of required environment variables for this claim to take precedence.
```

Comment:
Probably need to expand on this a bit—talk about fallback claims, behavior of an unreferenced variable if not treated as required, etc.

Reply:
I expanded on this, hope new definition is ok